src/Microsoft.AspNetCore.Razor.Language/Extensions/DesignTimeDirectivePass.cs

```diff
@@ -9,8 +9,9 @@ internal class DesignTimeDirectivePass : IntermediateNodePassBase, IRazorDirecti
     {
         internal const string DesignTimeVariable = "__o";
 
-        // This needs to run before other directive classifiers.
-        public override int Order => -10;
+        // This needs to run after other directive classifiers. Any DirectiveToken that is not removed 
+        // by the previous classifiers will have auto-generated design time support.
+        public override int Order => 100;
```

Comment:
Thinking too low. Hmm, since these are only ordered against other classifier passes I'm thinking this might make sense to be 1000 (`DefaultFeatureOrder`)